tests/tests_integration/test_api/test_extraction_pipelines.py

```diff
@@ -33,6 +36,28 @@ def new_extpipe(cognite_client):
     assert cognite_client.extraction_pipelines.retrieve(extpipe.id) is None
 
 
+@pytest.fixture
+def populated_runs(cognite_client: CogniteClient, new_extpipe: ExtractionPipeline) -> ExtractionPipelineRunList:
+    now = datetime_to_ms(datetime.now(timezone.utc))
+    a_year_ago = datetime_to_ms(datetime.now(timezone.utc).replace(year=datetime.now(timezone.utc).year - 1))
```

Comment:
Didn't you create a nice helper for this already 😉 
```suggestion
    now = datetime_to_ms(dt_now := datetime.now(timezone.utc))
    a_year_ago = datetime_to_ms(YearAligner.add_units(dt_now, -1))
```